thoth/package_extract/image.py

```diff
@@ -464,6 +465,24 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
+        return None
+    if os.path.islink(path):
+        readlink = os.readlink(path)
+        if len(readlink.split("/")) == 1:
+            next_link = os.path.join(os.path.dirname(path), readlink)
+        else:
+            if readlink[0] == "/":
+                next_link = os.path.join(root_path, readlink[1:])
+            else:
+                next_link = os.path.join(root_path, readlink)
+        return _get_absolute_link(root_path, next_link, iter + 1)
```

Comment:
And we recursively repeating until we stop at the point where it is not a symlink, but just a file and that is what would be returned.